region/brinstar/pink.json

```diff
@@ -1066,7 +1066,7 @@
                   ]
                 },
                 {
-                  "name": "CF Spore Spawn Skip",
+                  "name": "Big Pink Crystal Flash Standup",
```

Comment:
Needs to clarify you are shooting the super block
So `Crystal Flash Spore Spawn Skip` .  Standup is just a part of CFing.